PKGBUILDS/danctnix/mmsd-tng/PKGBUILD

```
@@ -0,0 +1,34 @@
+# Maintainer: Aren <aren@peacevolution.org>
+# Contributer: Caleb Fontenot <foley2431 at gmail dot com>
+
+pkgname=mmsd-tng
+pkgver=1.5
+pkgrel=1
+pkgdesc="Multimedia Messaging Service Daemon - The Next Generation"
+url="https://gitlab.com/kop316/mmsd"
+arch=('x86_64' 'armv7h' 'aarch64')
+license=("GPL2")
+provides=("mmsd")
+conflicts=("mmsd")
+depends=("c-ares" "mobile-broadband-provider-info" "libmm-glib" "libsoup" "libphonenumber")
+makedepends=("meson")
+install=./mmsd-tng.install
```

```suggestion
install=mmsd-tng.install
```
`./` is unnecessary.